### DISCUSSION TITLE
Unable to submit application, page keeps refreshing

### DISCUSSION THREAD

**Discussion by josephko91** [community/community#121149] [GitHub Education]
### Hi everyone,

I am trying to submit an application for Github Education but every time I press submit, the page just refreshes instead of the application submitting. I am uploading the picture of my University issued ID through the "upload" button, and I'm using Chrome on a laptop. On a related note, I was trying to take a picture directly (instead of uploading file) by using the web browser on my iPhone, but when I try to use the "back camera" instead of the front camera, if will not let me take the photo! I think this is a bug....?
How can I get this application submitted? Can someone help me out? Thanks!

**Comment by Alejand2r** [comment]
I have exactly the same problem, the page just keeps refreshing when i upload my student id 
Update: I just tried submitting on Safari and it worked out lol, so maybe try do it on Safari from your phone

**Comment by DavidSchmidt3** [comment]
I have also encountered this problem today, I have tried to submit 915 KB file, it looked ok, however after submit, page just refreshed. After inspecting the response, I have noticed response 413 content too large, suggesting that 1 MB file limit is actually too large to pass. After compressing the image further to something around 500 KB, it has worked. Someone should look at this, because you can't tell what is wrong and not sure, if application was actually submitted.

**Reply by marvin-oh** [reply]
The same thing had happened to me. I compressed the image to about 300KB and it was submitted successfully. Thank you.
